### ISSUE TITLE
Should users be able to plot partial dependence for any subset of classes in multiclass problems?

### ISSUE THREAD

**Issue by freddyaboulton** [alteryx/evalml#1565]
In #1554 @bchen1116 asked whether we should let users plot any subset of classes as opposed to just one or all:

> LGTM! One thing I think would be cool to have is to allow users to pass in a list of classes to create these plots for in multiclass scenarios. For instance, I, as a user, decided to create a multiclass pipeline with 20 possible target classes, I might want to plot partial dependencies for a subset of the classes only, rather than for 1 or all. Certainly not blocking, but wanted to bring that suggestion up for possible discussion.

https://github.com/alteryx/evalml/pull/1554#pullrequestreview-554019462

I'm on board. I don't think this should be a big lift from what we already have and I see it being useful. What do people think?

**Comment by angela97lin** [comment]
Yup, if we have the ability to plot for one or all, I don't see why not a subset! :+1:
